Rakefile

```diff
@@ -60,7 +60,7 @@ desc "Generate markdown docs for Bolt's core Puppet functions"
 task :docs do
   FileUtils.mkdir_p 'tmp'
   tmpfile = 'tmp/boltlib.json'
-  PuppetStrings.generate(PuppetStrings::DEFAULT_SEARCH_PATTERNS,
```

Comment:
Why this change? https://github.com/puppetlabs/puppet-strings/blob/7ab471857f957836759ea0e666568d60fa25b954/lib/puppet-strings.rb#L4-L11 seem ok?

Reply:
It includes `lib/**/*.rb` which results in a lot of warnings for files that we don't intend to write docs strings for. It just seems unnecessary when we already have to specify the actual path we want to build docs from.